earthpy/plot.py

```diff
@@ -70,33 +71,34 @@ def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
     return fig.colorbar(mapobj, cax=cax)
 
 
-# Function to plot all layers in a stack
 def plot_bands(
     arr, title=None, cmap="Greys_r", figsize=(12, 12), cols=3, extent=None
 ):
-    """Plot each layer in a raster stack read from rasterio in
-    (band, row , col) order as a numpy array. plot_bands will create an
-    individual plot for each band in a grid.
+    """Plot each layer in a raster stack read from rasterio as a numpy array.
```

Comment:
hey @mirob9363 it looks like you also removed some important information here - ie the order that the data should be in. Will you please keep this part of the docstring as it is?